presidio-image-redactor/presidio_image_redactor/qr_image_analyzer_engine.py

```diff
@@ -0,0 +1,62 @@
+from typing import List, Optional
+
+from presidio_analyzer import AnalyzerEngine
+
+from presidio_image_redactor.entities import ImageRecognizerResult
+from presidio_image_redactor.qr_recognizer import QRRecognizer
+from presidio_image_redactor.qr_recognizer import OpenCVQRRecongnizer
+
+
+class QRImageAnalyzerEngine:
```

Comment:
Can this class be inherited from `ImageAnalyzerEngine`? Just a question, to see if we can simplify the design instead of extending it to a new set of independent classes.

Reply:
Was thinking exactly the same, see below.

Reply:
Yes, it can be inherited from `ImageAnalyzerEngine`. My concern is that in this case, `QRImageAnalyzerEngine` will also inherit the logic of working with ocr tools not related to QR code recognition.

Reply:
Yes, that's my concern too. As the package is still in beta, we should (carefully) consider breaking backward compatibility. We'll do some thinking on this and get back to you. We can also have a quick design session together over video if you're interested.

Reply:
Yes, that sounds interesting. If you have time, we could do that.

Reply:
Sure. To avoid putting personal emails on GH, could you please email presidio@microsoft.com and we'll continue the discussion over email?